src/components/screen/Login.tsx

```diff
@@ -168,9 +169,12 @@ function Screen(props: Props): React.ReactElement {
       const { type, user } = await GoogleSignIn.signInAsync();
       if (type === 'success') {
         setGoogleUser(user);
+        Alert.alert('login:' + JSON.stringify(user));
+        onLogin();
       }
-    } catch ({ message }) {
-      Alert.alert('login: Error:' + message);
+    } catch (error) {
+      // Alert.alert('login: Error:' + message);
+      Alert.alert(error);
```

Comment:
Try to change this to `Alert.alert(error.message)` as it is in `facebook`.

Reply:
Alright. I'll try!